FordLabs/retroquest
# Add cypress test to test changing team password flow

## Overview
Adding cypress tests to test the password reset flow and reset board owners flow as best we can until we can test the actual email that is sent.

## Testing Instructions
1) Ensure that the password reset flow for when a user is not logged in is sufficiently tested.
2) Ensure that the password reset flow for when a user is logged in is sufficiently tested.
3) Ensure that the reset board owners flow for when a user is logged in is sufficiently tested.
4) Ensure that recovering team names by email is sufficiently tested.
